package.json

```diff
@@ -19,7 +19,7 @@
     "tslint": "^5.18.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-eslint-rules": "^5.4.0",
-    "typescript": "^3.5.3",
+    "typescript": "~3.4.0",
```

Comment:
@siddhant1 Why are we downgrading the version?

Reply:
I did that because Vega-Lite is not yet TS 3.5 ready.